api/nodes/serializers.py

```diff
@@ -555,3 +559,97 @@ def error_checker(self, data):
     def get_absolute_url(self, obj):
         #  Citations don't have urls
         raise NotImplementedError
+
+
+class DraftRegistrationSerializer(JSONAPISerializer):
+
+    schema_choices = list(ACTIVE_META_SCHEMAS)
```

Comment:
Yes, I can change this. I remember you saying something about both primary key and name, but I thought if I took the name they specified, and then looked up the version 2 schema for that name, we would be using the correct schema.

But I will change it so primary key is used.